src/battle-scene.ts

```diff
@@ -2032,29 +2032,11 @@ export default class BattleScene extends SceneBase {
     return filteredSpecies[randSeedInt(filteredSpecies.length)];
   }
 
-  generateRandomBiome(waveIndex: number): Biome {
-    const relWave = waveIndex % 250;
-    const biomes = getEnumValues(Biome).slice(1, getEnumValues(Biome).filter((b) => b >= 40).length * -1);
-    const maxDepth = biomeDepths[Biome.END][0] - 2;
-    const depthWeights = new Array(maxDepth + 1)
-      .fill(null)
-      .map((_, i: number) => ((1 - Math.min(Math.abs(i / (maxDepth - 1) - relWave / 250) + 0.25, 1)) / 0.75) * 250);
-    const biomeThresholds: number[] = [];
-    let totalWeight = 0;
-    for (const biome of biomes) {
-      totalWeight += Math.ceil(depthWeights[biomeDepths[biome][0] - 1] / biomeDepths[biome][1]);
-      biomeThresholds.push(totalWeight);
-    }
-
-    const randInt = randSeedInt(totalWeight);
-
-    for (const biome of biomes) {
-      if (randInt < biomeThresholds[biome]) {
-        return biome;
-      }
-    }
-
-    return biomes[randSeedInt(biomes.length)];
+  /**
+   * TODO: Rewrite this later for weighting?
+   */
+  generateRandomBiome(_waveIndex: number): Biome {
+    return Array.from(allBiomes.keys())[randSeedInt(allBiomes.keys.length)];
```

Comment:
Wait, what exactly is this doing? Just picking any random biome?
If so, this is simpler:
```suggestion
    return randSeedItem([...allBiomes.values()]);
```

Reply:
Should be the `keys()` rather than `values` since `allBiomes` is a map of Biome (the enum) to PEBiome (the class) and this function needs the enum but thanks, this looks simpler